README.md

```diff
@@ -5,12 +5,48 @@ The purpose of this project is to create new themes for Bizzabo website sections
 In `src` folder you will find a filed called `Section.js`.  
 This file is a template for the new section theme.  
 Please duplicate this file before you start.
+Inside the `props` object you'll find the relevant data for your section. Please find below section-specific instructions for your your development process.
+Inside the `Section.d.ts` file you can find TypeScript interfaces that describe the data structure for the section.
 
-You can find an example in the `ExampleSection.js` file.  
-It contains a real section that you can take inspiration from.  
-You can use the css classes that are used in the `ExampleSection.js`, but it's not a must.  
+You can find an example in the `examples` folder.  
+It contains a real sections that you can take inspiration from.  
+You can use the css classes that are used in the example sections, but it's not a must.  
 Please don't edit any other file other then your duplicated `Section.js` file.
 
+## Important classes
+There are several important classes that must to be included in the section markup:
+- `atom-sectionMain` should be on the top level wrapper of the section  
+- `element-${element.id}` should wrap every element in the section. (for example in the `people` section, `header` and `people` elements).
+- Please find another important classes that are section-specific.
+
+## Section-specific instruction
+### People
+Inside the section prop, you will find all the relevant data for the `people` section.
+`people` section elements are `people` and `header`.  
+ 
+ **NOTE:** The naming is a little bit confusing. `people` **section** contains two elements: `people` **element** and `header` **element**  
+
+ If `people` element contains `contactListId` prop, show the contacts, overwise, show the placeholder.  
+
+ In the `people` element you will find the contacts array.  
+ Each `contact` can be a speaker and than he will have a `speakerId` prop. In this case, the contact img should be wrapper with an A tag.  
+
+If the `contact` doesn't have image url (`thumbnailUrl`), show the placeholder instead.
+When hovering over an contact image, there is a small zoom effect. You can see it in the example section.
+
+The number of contacts to show in each view size:
+- Web view - show up to 4 contacts in a row.
+- Ipad view - show up to 3 contacts in a row.
+- Mobile view - show up to 1 contacts in a row.
+
```

Comment:
add also that each contact/person (depends how you call it here) should be aligned center